index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <header>My logo</header>
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+    <main class="main-section">
+      <h1>
+        I'm Juan<br />
+        Glad to see you!
+      </h1>
+      <h3>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </h3>
```

Comment:
- [ ] So nice :smile: :love_you_gesture: but for semantic correctness, this part of your code doesn't seem like a title or heading directly describing another section. Kindly consider using a `p` tag for this text. It's more of a paragraph, don't you agree? :wink: